pyproject.toml

```diff
@@ -161,6 +161,7 @@ ignore = [
     "TID252",  # if we're going to use relative imports, use them always
     "TRY003",  # good general advice but lint is way too aggressive
     "TRY301",  # sometimes raising exceptions inside try is the best flow
+    "UP040",   # Python 3.12 supports `type` alias kw, but mypy doesn't yet
```

Comment:
You should probably PR this up to the template...which, uh, I'm not sure where that comes from.